dev-requirements.txt

```diff
@@ -1,4 +1,5 @@
 django-debug-toolbar==1.5     # https://django-debug-toolbar.readthedocs.io/en/stable/changes.html
+setuptools==33.1.1
```

Comment:
On March 3 Taylor updated our pip deployment role version to 1.0.0 which by default installs pip `9.0.*` instead of `1.5.*`. That explains why we are now provisioning a newer pip version, which is good.

If installing requirements is still broken with the new pip verison, updating the tool versions may be the answer.